config/application.rb

```diff
@@ -21,7 +21,8 @@ module DonutRaiders
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
-
+    config.assets.enabled = true
+    config.assets.paths << Rails.root.join('/app/assets/fonts')
```

Comment:
You should be able to remove this once you are no longer storing the font locally!